.github/workflows/cd.yml

```diff
@@ -53,15 +53,14 @@ jobs:
           $WebClient.DownloadFile("https://github.com/deluge-torrent/gvsbuild-release/releases/download/latest/gvsbuild-py${{ matrix.python }}-vs16-${{ matrix.arch }}.zip","C:\GTK.zip")
           7z x C:\GTK.zip -oc:\GTK
           $env:Path = "C:\GTK\release;C:\GTK\release\bin;C:\GTK\release\lib;$env:Path"
-          python -m pip install --upgrade pip wheel
+          python -m pip install --upgrade pip
           $pycairopath = Get-Childitem –Path "C:\GTK\release\python\" -Include pycairo*.whl -File -Recurse -ErrorAction SilentlyContinue | select -expand FullName
           $PyGObjectpath = Get-Childitem –Path "C:\GTK\release\python\" -Include PyGObject*.whl -File -Recurse -ErrorAction SilentlyContinue | select -expand FullName
           pip install $pycairopath
```

Comment:
If we modify the run to use [folded style yaml](https://yaml-multiline.info/) this could be much cleaner:

```
run: >
   $WebClient = New-Object System.Net.WebClient
...

  pip install
  $pycairopath
  $PyGObjectpath
...
  pygame

  pip install -r requirements.txt
```

Remember to keep a space between commands

Reply:
imo "cleaner" is a personal opinion type thing so I am not going to try and cover this. :)